packages/front-end/pages/features/[fid].tsx

```diff
@@ -983,6 +983,8 @@ export default function FeaturePage() {
 
       <div className="mb-4">
         <h3>Test Feature Rules</h3>
+
+        {/* TODO: fix this. everything is null */}
```

Comment:
With the current rules I'd expect to see the value forced to on if id=1.

<img width="993" alt="image" src="https://github.com/growthbook/growthbook/assets/113377031/d61615d0-4b77-4489-a4f7-d140777c6189">